src/views/Reservations/Reservations.css

```diff
@@ -0,0 +1,86 @@
+:root {
```

Comment:
You might need to adjust the styling (colors, fonts, font-sizing, button etc.) on Reservtions a little bit later - for it to be more in-line with the rest of the app. Although for now it looks very nice on it's own 😛